source/joker.c

```diff
@@ -281,35 +282,49 @@ bool joker_object_score(JokerObject *joker_object, Card* scored_card, enum Joker
     *retrigger = joker_effect.retrigger;
     // joker_effect.message will have been set if the Joker had anything custom to say
 
-    int cursorPosX = fx2int(joker_object->sprite_object->x) + 8; // Offset of 16 pixels to center the text on the card
+    int cursorPosX = 8; // Offset of one tile to better center the text on the card
+    int cursorPosY = 0;
+    if (joker_event != JOKER_EVENT_ON_CARD_HELD)
+    {
+        cursorPosX += fx2int(joker_object->sprite_object->x);
+        cursorPosY = JOKER_SCORE_TEXT_Y;
+    }
+    else
+    {
+        // display the text on top of the card instead of below the Joker for Held Cards effects
+        // scored_card cannot be NULL here because of the joker event
+        cursorPosX += fx2int(card_object->sprite_object->x);
+        cursorPosY = HELD_CARD_SCORE_TEXT_Y;
+    }
```

Comment:
```suggestion
    if (joker_event == JOKER_EVENT_ON_CARD_HELD)
    {
         // display the text on top of the card instead of below the Joker for Held Cards effects
        // scored_card cannot be NULL here because of the joker event
        cursorPosX += fx2int(card_object->sprite_object->x);
        cursorPosY = HELD_CARD_SCORE_TEXT_Y;
    }
    else
    {
        cursorPosX += fx2int(joker_object->sprite_object->x);
        cursorPosY = JOKER_SCORE_TEXT_Y;
    }
```

Switch the cases, the double negative here makes it confusing to read